client/plots/controls.overlay.js

```diff
@@ -33,7 +33,7 @@ class Overlay {
 	}
 	initPill() {
```

Comment:
todo later/note: the `defaultQ4fillTW` will go away with the `tw` refactor, should pass a `usecase` option instead and the `tw` object will know what the default q's are for a given `usecase`. This `usecase` will also match the `termsettingInit()` `opts.usecase`.